autopush/utils.py

```diff
@@ -301,6 +302,8 @@ class WebPushNotification(object):
     # Whether this notification should follow legacy non-topic rules
     legacy = attrib(default=False)  # type: bool
 
+    expry = attrib(default=MAX_EXPRY)  # type: int
```

Comment:
Is this expry attribute here even necessary? We seem to always fill the expry headed towards the database from the ttl attribute. Only thing I see this doing is propagating it over the connection node via JSON, but it's not even used there

Reply:
Actually, going to leave it. We use it for several tests and it identifies as a keyword parameter.